python/docs/_static/copybutton.js

```diff
@@ -0,0 +1,66 @@
+// Copyright 2014 PSF. Licensed under the PYTHON SOFTWARE FOUNDATION LICENSE VERSION 2
+// File originates from the cpython source found in Doc/tools/sphinxext/static/copybutton.js
```

Comment:
Given its source, is there any way to get this to render automatically with sphinx or anything? not sure that even makes sense, just checking if this is something that has to be copied into the project or not.

Reply:
@srowen I didn't find any other way to get this rendered with sphinx. According to my [references](https://github.com/apache/spark/pull/24456#issuecomment-486600616), it is used in the same way by other libraries like scipy, and scikit-learn.